src/uri.rs

```diff
@@ -186,6 +182,33 @@ impl Rsync {
         &self.as_str()[self.module_start..(self.path_start - 1)]
     }
 
+    /// Returns the URI’s module.
+    ///
+    /// The module is identical to the a URI with an empty path.
+    pub fn module(&self) -> &str {
+        &self.as_str()[..self.path_start]
```

Comment:
This definition of `module()` differs from that which was offered by RsyncModule::module() which was defined as being the module name, not the leading part of the Rsync URI upto-and-including the trailing slash after the module name (it was explicitly defined as `self.uri.module_name()`). This is a change in behaviour while the changelog update gives the impression that the functionality has been moved into the existing Rsync type without change. Is this intentional? Was the RsyncModule::module() fn considered to be wrong and this this is now right? Should the changelog update note this difference in some way?

Reply:
Previously, the term was a bit overloaded: module was used as both the URI identifying the module, i.e., the combination of authority and module name – that’s what `RsyncModule` actually contains, and the name of the module itself within the authority. I decided to clean this up: _module_  now refers the global identifier of the module (i.e., what `RsyncModule` used to represent) while the identifier of a module within the same authority is now the _module name_.

This could indeed be clarified in the changelog.